### PR TITLE
Finished assessment.

### PR DESCRIPTION
**Pull request by kaylal12**

### CODE REVIEW

**Comment by kaylal12** [comment]
Misread Question 5 and made it print out numbers from 1 to 10.

File: README.md [hunk 1]
```diff
@@ -16,7 +16,11 @@ You should save your answers in this README.md file, except for your answer to q
 
 Please list the JavaScript primitives and give one example of each.
 
-<!-- Replace this comment with your answer -->
+Number: -0, NaN, Infinity, 12
```
Comment (GA-MEB) [inline README.md:19]: Very nice! Yes, `NaN` is, oddly enough, a number.

File: README.md [hunk 2]
```diff
@@ -28,7 +32,7 @@ c = c + d;
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
 
-<!-- Replace this comment with your answer -->
+c = 10, in the first two lines the "=" is used as an assignment operator, meaning it gives var c the value of 5 and var d the value of 2. In the 3rd line, the addition sign is used to add the values of var c and d together and after that has been done, the equal sign will assign c to the sum.
```
Comment (GA-MEB) [inline README.md:35]: Right reasoning, wrong answer; c = 7.

File: README.md [hunk 3]
```diff
@@ -53,7 +57,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+In the first line, the variable weather is undefined. In the second line, there is one equal sign used, which means it has been assigned the value of "sunny". In the third line, it should evaluate to "true" because the use of three equal signs means that is a strict equality (meaning is it trying to see if the value of weather is equal to "sunny").
```
Comment (GA-MEB) [inline README.md:60]: Yes, this is right. It should be noted, though, that `weather = "sunny";` is also an expression, and it also evaluates to a value (in this case, "sunny").

File: assessment.js [hunk 4]
```diff
@@ -1,8 +1,9 @@
...
 var ask = require('./ask.js');
 
-var answer = 'not empty';
+var i = 1;
 
-while (answer !== '' && answer !== 'SeCrEt') {
-  answer = ask("Guess my secret? ");
+while (i <= 10) {
+  console.log(i);
+  i++;
```
Comment (GA-MEB) [inline assessment.js:8]: Yeah, that's not the right answer. :)

Feel free to resubmit for this question - I'd be happy to give you feedback.

**Comment by GA-MEB** [comment]
3 of 5

Not bad, but I think you can definitely solve number 5. Why don't you give it another shot?

**Pull request closed without merging**